packages/server-socket/src/__snapshots__/command-processor.spec.js.snap

```diff
@@ -16,6 +16,7 @@ Object {
     "labels": undefined,
     "title": "Change a title",
     "todoId": "id-1",
+    "trackedTimes": undefined,
```

Comment:
If "default" is "undefined", couldn't we remove / ignore trackedTimes here?

Reply:
no, because mongoose would give us an empty array then. which overwrites the real values